backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/button/internal/BpkButtonImpl.kt

```diff
@@ -74,16 +74,19 @@ internal fun BpkButtonImpl(
                 .requiredHeight(size.minHeight),
             interactionSource = interactionSource,
             colors = ButtonDefaults.buttonColors(
-                backgroundColor = type.backgroundColor(interactionSource),
+                containerColor = type.backgroundColor(interactionSource),
                 contentColor = type.contentColor(interactionSource),
-                disabledBackgroundColor = if (loading) type.loadingBackgroundColor() else type.disabledBackgroundColor(),
+                disabledContainerColor = if (loading) type.loadingBackgroundColor() else type.disabledBackgroundColor(),
```

Comment:
Button will have some screenshots changed, and this is because in MD2 button the alpha wasn't applied.
This affects only the loading state in LinkOnDark as it uses linkOnDarkPressedForeground which has an alpha channel.

The reason for that is MD2 button was forcing alpha to be 1f and using ContentAlpha instead.
The new version is correct.